File: benchmarks/live_suite/fib.c
#include <stdio.h>
#include <stdint.h>

uint64_t fib(uint64_t n) {
    if (n <= 1) return n;
    return fib(n - 1) + fib(n - 2);
}

int main(void) {
    uint64_t res = fib(40);
    printf("Result: %llu\n", (unsigned long long)res);
    return 0;
}

File: benchmarks/live_suite/fib.go
package main

import "fmt"

func fib(n uint64) uint64 {
	if n <= 1 {
		return n
	}
	return fib(n-1) + fib(n-2)
}

func main() {
	res := fib(40)
	fmt.Printf("Result: %d\n", res)
}
